edudoc/disciplinary-crash-course.md

```diff
@@ -0,0 +1,96 @@
+## WCA Disciplinary Crash Course
+
+# Disciplinary
```

Comment:
Since this isn't the title of the document, this should have two hashes: `## Disciplinary`.

On another note, I'm not sure that "Disciplinary" is a great title for this section or document? It's not really descriptive about what this section is about. If it's just an intro, you can get away with not having a section either.

Reply:
We are still working on some names, that was mostly a place holder :p

I was thinking something to the effect of "Guidelines for handling behavioral issues and cheating".